src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -193,7 +193,7 @@ export class ExposureNotificationService {
   }
 
   private async init() {
-    const exposureStatus = JSON.parse((await this.storage.getItem(EXPOSURE_STATUS)) || 'null');
+    const exposureStatus = JSON.parse((await this.secureStorage.get(EXPOSURE_STATUS).catch(() => null)) || 'null');
```

Comment:
@henrytao-me - why is there both null value and null string in this? 

In one case, `exposureStatus = null`, and in the other case, `exposureStatus = 'null'`.

Reply:
The bracket is the key. 
- The 1st part is `(await this.secureStorage.get(EXPOSURE_STATUS).catch(() => null))` which run async and return string | null. 
- The 2nd part will make sure it fallbacks to `'null'` string for `JSON.parse`. 

That means `.catch(() => 'null')` will have no different in code execution.